src/SFA.DAS.EAS.Web/FileDownloads/Payments/ExcelTransactionFormatter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.EAS.Domain.Interfaces;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+using SFA.DAS.EAS.Web.ViewModels;
+
+namespace SFA.DAS.EAS.Web.FileDownloads.Payments
+{
+    public class ExcelTransactionFormatter : ITransactionFormatter
+    {
+        private readonly IExcelService _excelService;
+
+        public ExcelTransactionFormatter(IExcelService excelService)
+        {
+            _excelService = excelService;
+        }
+
+        public byte[] GetFileData(List<TransactionDownloadLine> transactions)
+        {
+            var excelRows = new List<string[]>();
+
+            excelRows.Add(GetHeaderRow());
+
+            foreach (var transaction in transactions)
+            {
+                excelRows.Add(new[]
+                {
+                    transaction.DateCreated.ToString(), transaction.TransactionType,
```

Comment:
Using just ToString() means we cannot guarantee the format of the date (as the culture on the current thread will be used) - would it be better to use the invariant culture so we can guarantee format or perhaps use a fixed date format?

The same goes for elsewhere where dates use string interpolation. Int types ar eless of a problem but these should also be invariant.

Reply:
This one fell through the net, I'll update it now. We generally do use formats but I wouldn't be surprised if there are a few missing in the whole of the code. I was waiting to ask David what format he was using as his code wasn't finished but forgot to do this and didn't go back to change it. Given its being displayed we may have wanted to use currentculture to reflect what the server was setup for but I think a fixed format is better as we can then use a format excel supports easily.